### ISSUE TITLE
Question: rule outputs are confusing

### ISSUE THREAD

**Issue by marcolarosa** [Language-Research-Technology/ldac-profile#6]
The validator reports warnings like:
```
    {
      entity: './',
      property: '@type',
      message: "Property `iso639-3` is not defined in the crate's context",
      clause: 'MUST have a `@type` attribute that that includes in its values `Dataset` and either `RepositoryCollection` or `RepositoryObject`'
    },
```

So I know which entity it refers to but the message relates to the property not being defined in the context when the clause talks about the @type being incorrect (which it's not).

**Comment by ptsefton** [comment]
Good point. This is a limitation of the current design where rules have to be tied to a property so the test is running under the @type prop -- we will have to fix this.

----------------------------

Dr Peter Sefton
Senior Technical Advisor, School of Languages and Culture
Mobile: 0404 096 932

________________________________
From: Marco La Rosa ***@***.***>
Sent: Wednesday, December 7, 2022 15:53
To: Language-Research-Technology/ldac-profile ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Language-Research-Technology/ldac-profile] Question: rule outputs are confusing (Issue #6)


The validator reports warnings like:

    {
      entity: './',
      property: ***@***.***',
      message: "Property `iso639-3` is not defined in the crate's context",
      clause: 'MUST have a ***@***.***` attribute that that includes in its values `Dataset` and either `RepositoryCollection` or `RepositoryObject`'
    },


So I know which entity it refers to but the message relates to the property not being defined in the context when the clause talks about the @type<https://github.com/type> being incorrect (which it's not).
